clients/admin-ui/src/features/system/system-form-declaration-tab/PrivacyDeclarationDisplayGroup.tsx

```diff
@@ -34,52 +34,56 @@ const PrivacyDeclarationRow = ({
   const { isOpen, onOpen, onClose } = useDisclosure();
   return (
     <>
-      <Box px={6} py={4} data-testid={`row-${declaration.data_use}`}>
-        <HStack>
-          <LinkBox
-            onClick={() => handleEdit(declaration)}
-            w="100%"
-            h="100%"
-            cursor="pointer"
-          >
-            <LinkOverlay>
+      <LinkBox
+        w="100%"
+        h="100%"
+        cursor="pointer"
+        _hover={{ backgroundColor: "gray.50" }}
+      >
+        <Box px={6} py={4} data-testid={`row-${declaration.data_use}`}>
+          <HStack>
+            <LinkOverlay onClick={() => handleEdit(declaration)}>
```

Comment:
for full accessibility support, include these properties here
```suggestion
            <LinkOverlay onClick={() => handleEdit(declaration)} role="button" tabindex="0">
```